File: Backend/Entities/Character.cs
using System.ComponentModel.DataAnnotations.Schema;
using API.Entities;

namespace Backend.Entities
{
    [Table("Characters")]
    public class Character
    {
        public int Id { get; set; }

        public int Level { get; set; } = 1;

        public int HpMax { get; set; } = 100;

        public int HpCurrent { get; set; } = 100;

        public int XpMax { get; set; } = 100;

        public int XpCurrent { get; set; } = 0;

        public int Damage { get; set; } = 5;

        public int Accuracy { get; set; } = 1000;

        public int Armour { get; set; } = 0;

        public int Evasion { get; set; } = 0;

        public int CritChance { get; set; } = 1;

        public string PublicId { get; set; }

        public AppUser AppUser { get; set; }

        public int AppUserId { get; set; }
    }
}

File: NodeBackend/models/character.js
const mongoose = require('mongoose');

const characterScheme = mongoose.Schema({
    level: {type: Number, require: true},
    hpMax: {type: Number, require: true},
    hpCurrent: {type: Number, require: true},
    xpMax: {type: Number, require: true},
    xpCurrent: {type: Number, require: true},
    damage: {type: Number, require: true},
    accuracy: {type: Number, require: true},
    armour: {type: Number, require: true},
    evasion: {type: Number, require: true},
    critChance: {type: Number, require: true},
});

module.exports = mongoose.model('Character', characterScheme);